JavaHackerrank/Find_The_Running_Median.java

```diff
@@ -0,0 +1,74 @@
+import java.io.*;
```

Comment:
Problem link : https://www.hackerrank.com/challenges/find-the-running-median/problem